pandas/tests/arrays/categorical/test_cumulative.py

```diff
@@ -0,0 +1,49 @@
+"""
+Tests for Ordered Categorical Array cumulative operations.
+"""
+
+import numpy as np
+import pytest
+
+import pandas as pd
+import pandas._testing as tm
+
+
+class TestAccumulator:
+    @pytest.mark.parametrize(
+        "method, input, output",
+        [
+            ["cummax", [1, 2, 1, 2, 3, 3, 2, 1], [1, 2, 2, 2, 3, 3, 3, 3]],
+            ["cummin", [3, 2, 3, 2, 1, 1, 2, 3], [3, 2, 2, 2, 1, 1, 1, 1]],
+        ],
+    )
+    def test_cummax_cummin_on_ordered_categorical(self, method, input, output):
+        # GH#52335
+        result = pd.Categorical(input, ordered=True)._accumulate(method)
```

Comment:
We don't need to test `_accumulate` specifically, the tests in `pandas/tests/series/test_cumulative.py` are sufficient

Reply:
Removed pandas/tests/arrays/categorical/test_cumulative.py